src/Handler/Table/BadExportFilterParametersException.php

```diff
@@ -24,9 +25,9 @@ public function __construct(string $message, int $code = self::ERR_VALIDATION, ?
     /**
      * @throws self
      */
-    public static function handleWrongTypeInFilters(BadRequestException $e): void
+    public static function handleWrongTypeInFilters(BigqueryException|BadRequestException $e): void
     {
-        if ($e->getCode() === 400 && str_contains($e->getMessage(), 'No matching signature for operator ')) {
```

Comment:
@zajca tbh nevim proc tam byl tento check na 400. je to za tebe takto ok? tldr potrebuju, aby to sezralo `BigqueryException|BadRequestException`

Reply:
Jo to je cajk bych řekl.